src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppECSFargate.recipe

```diff
@@ -702,6 +702,15 @@
                             "Value": "Path"
                         }
                     ]
+                },
+                {
+                    "Id": "InternetFacing",
+                    "Name": "Internet-Facing",
+                    "Description": "Should the load balancer have an internet-routable address? Internet-facing load balancers can route requests from clients over the internet. Internal load balancers can route requests only from clients with access to the VPC.",
+                    "Type": "Bool",
+                    "DefaultValue": true,
+                    "AdvancedSetting": false,
+                    "Updatable": false
```

Comment:
Left this at `false` because of https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/780 during redeployment